.env.test.local

```diff
@@ -0,0 +1,2 @@
+REACT_APP_SUPABASE_URL=https://ezwbsacoojmonmiqffad.supabase.co
```

Comment:
We should not be uploading these to github. Try to double check that the files your env variables live in are inside of your .gitingore.